fennel/internal_lib/to_proto/to_proto.py

```diff
@@ -464,6 +464,11 @@ def feature_to_proto_as_input(f: Feature) -> fs_proto.Input:
 def _extractor_to_proto(
     extractor: Extractor, fs: Featureset, fs_obj_map: Dict[str, Featureset]
 ) -> fs_proto.Extractor:
+    extractor_name = (
+        extractor.name
+        if extractor.extractor_type == ExtractorType.PY_FUNC
+        else "Auto Generated"
+    )
```

Comment:
dont autogenerated extractors not have a name ? _fennel_alias .. or _fennel_lookup ?

Reply:
yup they do have, moving the error ton that.